packages/vite/src/node/ssr/ssrModuleLoader.ts

```diff
@@ -137,7 +137,6 @@ async function instantiateModule(
 
   const resolveOptions: NodeImportResolveOptions = {
     mainFields: ['main'],
-    browserField: true,
```

Comment:
I'm not sure if this was a mistake, the `browserField` is used in SSR resolving.

Reply:
Before #10071, Vite used browser field even with `ssrModuleLoader` and I wanted to preserve that behavior to avoid a breaking change. I think this change makes sense.